osbs/build/build_request.py

```diff
@@ -795,7 +795,14 @@ def set_arg(arg, value):
         if use_auth is not None:
             set_arg('use_auth', use_auth)
 
-        if self.spec.prefer_schema1_digest.value is not None:
+        # arrangement version 4 and later deals with pulp differently than
+        # earlier arrangement versions.
+        # < 4: set prefer_schema1_digest according to config
+        # >=4: set report_multiple_digests=True when pulp in use
+        if (self.spec.arrangement_version.value >= 4 and
+                self.spec.pulp_registry.value):
+            set_arg('report_multiple_digests', True)
```

Comment:
I thought we wanted to do this only when pulp supports schema 2?

Reply:
`koji_import` will remove any that didn't make it to the repository: https://github.com/projectatomic/atomic-reactor/pull/822/files#diff-15d5e9d9641101604342ff796e5a92c2R244